resources/splunk_install.rb

```diff
@@ -0,0 +1,219 @@
+# Encoding: utf-8
+
+class SplunkInstall < ChefCompat::Resource
+  include PlatformHelpers, PathHelpers
+
+  resource_name :splunk_install
+
+  property :name, String, name_property: true, identity: true
+  property :package, [:splunk, :universal_forwarder], required: true
+  property :service_user, String, default: 'splunk'
```

Comment:
Hm...good point... Do we do anything with the user in this resource other than ensure it's created? Do we need it for install time on Windows (I don't think I see it being used)? Or could user / service management for an installation be a different resource or different action? (action :enable / :disable to manage the service?)